rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/util/CovidRDTJsonFormUtils.java

```diff
@@ -67,7 +68,17 @@ private void  prePopulateSampleCollectionFormFields(JSONObject field, String uni
         }
         // pre-populate respiratory sample id field
         if (COVID_SAMPLE_ID.equals(field.getString(KEY))) {
-            field.put(VALUE, uniqueID);
+            field.put(JsonFormConstants.VALUE, uniqueID);
+        }
+
+        // pre-populate the patient detail unique id
+        if (CovidConstants.FormFields.PATIENT_DETAIL.equals(field.getString(KEY))) {
```

Comment:
After making this change https://github.com/onaio/rdt-standard/pull/456/files#r476344796, this block will not be necessary.

Reply:
fixed.